### ISSUE TITLE
`.response.created.unix_epoch_time` was `string` type, not `number`

### ISSUE THREAD

**Issue by kennedy** [ejstreet/terraform-provider-omglol#3]
Following example:

```terraform
data omglol_account_info this {}
```

I got this error:

```
json: cannot unmarshal number into Go struct field .response.created.unix_epoch_time of type string
```

my system is:

apple m1 pro
terraform 1.3.9
omg.lol provider v0.0.2

**Comment by kennedy** [comment]
fixed in `v0.1.0`

**Comment by ejstreet** [comment]
Glad it's fixed 😊
